app/views/layouts/_nav_menu.html.erb

```diff
@@ -18,7 +18,7 @@
     <li>
       <a href="<%= mod_path %>" class="c-link c-link--block trusted-visible-block"><%= t("views.main.nav.moderator_center") %></a>
     </li>
-    <li>
+    <li class="js-policy-Article-create">
```

Comment:
Is it convention not to have `id` attributes on `<li>` elements? 🤔

Reply:
By convention, we usually want to avoid `id` elements.  It's easy for those to collide with one another.

Reply:
Can we avoid capital letters in class names? It's not really "wrong" but it doesn't follow any of the _unwritten_ naming conventions we've been following :)

Reply:
I want to add a bit of context around IDs vs classes, as the overall approach is far from standardised in our codebase. However as a base rule:

If an ID is used there must only ever be one usage of it on a page. As @jeremyf says, that means we need to be mindful of collisions. I might prefer an ID in the case of e.g. targeting the button which triggers the header profile menu navigation, since there's only ever going to be one of it. In this case, using a class name allows us to extend in future and say "these 5 things adhere to this policy"

Reply:
(also agree class names should be 100% lowercased)